### PR TITLE
use checksum everywhere instead of lowercase (branched from master)

### PR DESCRIPTION
**Pull request by andytcf**
Fixes the issue where the update in the snapshot strategy does not match ```.toLowerCase()``` to addresses anymore and instead uses the ethers util ```ethers.utils.getAddress() // checksum check``` instead that is more reliable and consistent

### CODE REVIEW

File: mutations/gov/useSignMessage.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ const useSignMessage = () => {
 		async (payload: SignaturePayload) => {
 			const version = '0.1.3';
 			let msg: any = {
-				address: walletAddress?.toLowerCase(),
+				address: ethers.utils.getAddress(walletAddress ?? ''),
```
Comment (clementbalestrat) [inline mutations/gov/useSignMessage.ts:56]: can you confirm `ethers.utils.getAddress('')` is not going to return an error?

**Review by clementbalestrat — CHANGES_REQUESTED** [review]
LGTM Candy. Just a minor comment about using `getAddress()` with an empty string. Also History.tsx has a warning with a missing React Hook. Please make sure to run `npm run lint` and see if everything is fine.

Also remove the `package-lock.json` changes.

File: mutations/gov/useSignMessage.ts (hunk 1 above)
Comment (andytcf) [reply mutations/gov/useSignMessage.ts:56]: @clementbalestrat good pick up

**Comment by andytcf** [comment]
@clementbalestrat actioned those changes requested

**Pull request merged**